pkg/abi/nvgpu/ctrl.go

```diff
@@ -90,12 +90,20 @@ type NV0080_CTRL_FIFO_GET_CHANNELLIST_PARAMS struct {
 
 // From src/common/sdk/nvidia/inc/ctrl/ctrl0080/ctrl0080gpu.h:
 const (
+	NV0080_CTRL_CMD_GPU_GET_CLASSLIST              = 0x800201
 	NV0080_CTRL_CMD_GPU_GET_NUM_SUBDEVICES         = 0x800280
 	NV0080_CTRL_CMD_GPU_QUERY_SW_STATE_PERSISTENCE = 0x800288
 	NV0080_CTRL_CMD_GPU_GET_VIRTUALIZATION_MODE    = 0x800289
 	NV0080_CTRL_CMD_GPU_GET_CLASSLIST_V2           = 0x800292
 )
 
+// +marshal
+type NV0080_CTRL_CMD_GPU_GET_CLASSLIST_PARAMS struct {
+	NumClasses	uint32
+	Pad   [4]byte
```

Comment:
nit: `_ uint32` seems to be the convention for such padding fields.

Reply:
Oh, I see now that searching for `_ uint32` shows it used in a few other files, but in this file there's only `Pad  [x] byte` style usage. What's the reason for the discrepancy in this file?

Reply:
Hmm it seems to be because some of the paddings are of odd number of bytes (3 bytes, 6 bytes, etc). I guess either is fine, will leave it up to you.